Detectors/PHOS/reconstruction/src/AltroDecoder.cxx

```diff
@@ -65,8 +65,14 @@ void AltroDecoder::readChannels(const std::vector<uint32_t>& buffer, CaloRawFitt
       if (currentword != 0) {
         LOG(ERROR) << "Channel header mark not found, header word " << currentword;
         short fec = header.mHardwareAddress >> 7 & 0xf; //try to extract FEE number from header
-        if (fec < 0 || fec > 13) {
-          fec = kGeneralSRUErr;
+        short branch = header.mHardwareAddress >> 11 & 0x1;
+        if (branch < 0 || branch > 1) {
+          fec = kGeneralTRUErr + kGeneralSRUErr;
```

Comment:
Result of anything & 0x1 is either 0 or 1, so the condition of this `if` is always false. What is its point?